bin/martinize2

```diff
@@ -618,16 +620,60 @@ def entry():
     )
     go_group = parser.add_argument_group("Virtual site based GoMartini")
     go_group.add_argument(
-        "-govs-includes",
+        "-go",
         action="store_true",
         default=False,
-        help="Write include statements to use Vitrual Site Go Martini.",
+        help="Apply the Martini Go model as structural bias.",
     )
     go_group.add_argument(
         "-govs-moltype",
+        dest="govs_moltype",
         default="molecule_0",
         help=("Set the name of the molecule when using " "Virtual Sites GoMartini."),
     )
```

Comment:
Nuke it!
In with the new, out with the old!

Reply:
Turns out this flag is very important! Currently, it is not possible to run the Go map for larger(ish) complexes. Thus, one common workflow routine consists of multiple martinize2 runs for different assemblies of the large complex. Each of these assemblies get's a Go bias potential. Then everything is combined together. Having the moltype flag allows the user to set the molecule name while looping over these. If removed it becomes super annoying because all prefixes of the Go-type have to be changed. So we very much want to keep this. 

I think we might even want to allow setting of a molecule name prefix in general